Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -155,6 +167,37 @@ click: function () { closeClicked.raiseEvent(this); }');
         },
 
         /**
+         * Gets or sets the security settings for iFrames generated by the Infobox.
+         * This will disable sandboxing for the iFrame, and allow HTML to be in the title.
+         * Only do this if you are in complete control of the data that will be loaded into the InfoBox,
+         * otherwise it is a security issue.
+         * @memberof InfoBox.prototype
+         *
+         * @type {Boolean}
+         */
+        disableSecurity : {
+            get : function() {
+                return this._disableSecurity;
+            },
+            set : function(value) {
+                this._disableSecurity = value;
+                var e = this._element.getElementsByClassName('cesium-infoBox-title')[0];
+                if (value) {
+                    e.removeAttribute('data-bind');
+                    e.setAttribute('data-bind', 'html: titleText');
+                    this._frame.removeAttribute('sandbox');
+                }
+                else {
+                    e.removeAttribute('data-bind');
+                    e.setAttribute('data-bind', 'text: titleText');
+                    this._frame.setAttribute('sandbox', 'allow-same-origin allow-popups allow-forms');
+                }
+                knockout.cleanNode(this._element);
```

Comment:
You could have an observable to the viewModel that tracks whether `disableSecurity` is true or false. You may only need the `if/else` bindings in this case. When its true, render the element with `titleHtml` with the `html` binding. Otherwise render the element with `titleText` and the `text` binding. When `titleText` or `titleHtml` values are updated, the viewModel takes care of it thanks to knockout. Does that make sense?